cogs/tags.py

```diff
@@ -394,7 +394,7 @@ async def make(self, ctx):
         create command.
         """
 
-        await ctx.send('Hello. What would you like the name tag to be?')
+        await ctx.send('Hello. What would you like the tag\'s name to be?')
```

Comment:
It's missing a few words, "name of the tag" or "tag's name" works.

```suggestion
        await ctx.send("Hello. What would you like the tag's name to be?")
```